modules/apps/change-tracking/change-tracking-rest-impl/rest-openapi.yaml

```diff
@@ -274,6 +274,34 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/CTCollection"
             tags: ["CTCollection"]
+    "/ct-collections-by-class":
```

Comment:
Brian is going to complain about this path. I think it should be something like 
`/ct-collections/by-model-class-name-id/{modelClassNameId}/by-model-class-pk/{modelClassPK}`